evap/student/tests/test_live.py

```diff
@@ -0,0 +1,157 @@
+        baker.make(Question, questionnaire=bottom_general_questionnaire, order=3, type=QuestionType.GRADE)
+
+        baker.make(
+            Contribution,
+            contributor=contributor1,
+            questionnaires=[contributor_questionnaire],
+            evaluation=self.evaluation,
+        )
+        baker.make(
+            Contribution,
+            contributor=contributor2,
+            questionnaires=[contributor_questionnaire],
+            evaluation=self.evaluation,
+        )
+
+        self.evaluation.general_contribution.questionnaires.set(
+            [top_general_questionnaire, bottom_general_questionnaire]
+        )
+
+    def _get_elements(self):
+        WebDriverWait(self.selenium, 10).until(
+            expected_conditions.visibility_of_element_located((By.ID, "text_results_publish_confirmation_top"))
+        )
+        return {
+            "top": self.selenium.find_element(By.ID, "text_results_publish_confirmation_top"),
+            "bottom": self.selenium.find_element(By.ID, "text_results_publish_confirmation_bottom"),
+            "bottom_card": self.selenium.find_element(By.ID, "bottom_text_results_publish_confirmation_card"),
+        }
+
+    def test_checking_top_confirm_checkbox_checks_and_hides_bottom(self):
+        self._setup()
+        self._login(self.voting_user1)
+
+        self.selenium.get(self.live_server_url + reverse("student:vote", args=[self.evaluation.pk]))
+
+        elements = self._get_elements()
+        elements["top"].click()
+
+        assert elements["bottom"].is_selected()
+        assert "d-none" in elements["bottom_card"].get_attribute("class")
+
+    def test_checking_bottom_confirm_checkbox_check_top_but_keeps_bottom_visible(self):
+        self._setup()
+        self._login(self.voting_user1)
+
+        self.selenium.get(self.live_server_url + reverse("student:vote", args=[self.evaluation.pk]))
+
+        elements = self._get_elements()
+        elements["bottom"].click()
+
+        assert elements["top"].is_selected()
+        assert "d-none" not in elements["bottom_card"].get_attribute("class")
+
+    def test_resolving_submit_errors_clears_warning(self):
+        self._setup()
+        self._login(self.voting_user1)
+
+        self.selenium.get(self.live_server_url + reverse("student:vote", args=[self.evaluation.pk]))
+        WebDriverWait(self.selenium, 10).until(
+            expected_conditions.presence_of_element_located((By.ID, "vote-submit-btn"))
+        ).click()
+
+        for row in self.selenium.find_elements(By.CSS_SELECTOR, "#student-vote-form .row"):
+            try:
+                checkbox = row.find_element(By.CSS_SELECTOR, "input[type=radio][value='2'] + label.choice-error")
+            except NoSuchElementException:
+                continue
+            checkbox.click()
+
+            assert len(row.find_elements(By.CSS_SELECTOR, ".choice-error")) == 0
+
+    def test_skip_contributor(self):
```

Comment:
This check was asserting that the card collapsed before in the TS implementation. I think the check should be possible with the new setup as well, or is there some issue with that?